### DISCUSSION TITLE
Wildcard character for Subscriber attributes

### DISCUSSION THREAD

**Discussion by ravi-prakash-v** [beckn/protocol-specifications#331] [Core Specification]
When a subscriber wants to operate in all domains, or in all cities, should we use the wildcard character `*` in the field instead of leaving it blank or null?

For example,

```
{
    "domain" : "mobility",
    "city" : "*",
    "country" : "IND",
    ...
}
```

represents a Subscriber that operates in all cities across India.

**Comment by venkatramanm** [comment]
Not passing the attribute 'city' is enough. Leave how it's persisted or handled to registry implementations.
